elements/bulbs-lightbox/bulbs-lightbox.js

```diff
@@ -0,0 +1,36 @@
+import {
+  BulbsHTMLElement,
+  registerElement,
+} from 'bulbs-elements/register';
+
+class BulbsLightBox extends BulbsHTMLElement {
+
+  attachedCallback () {
+
+    if (this) {
+
+      var overlay = this.querySelector('.bulbs-lightbox-overlay');
+      
+      function toggleOverlay() {
+        if(this.classList.contains('active')) {
+          this.classList.remove('active');
+        }
+        else if(!this.classList.contains('active')){
+          this.classList.add('active');
+        }
+      }
+
+      // Close overlay with Esc key
+      $(document).keyup(function(e) {
+        if (e.keyCode == 27) overlay.classList.remove('active');
+      });
+
+      this.addEventListener('click', toggleOverlay);
```

Comment:
I think we ought to translate these event listeners to be more consistent with the bulbs-elements house style.

Something like:

```js
document.addEventListener('keyup', this.handleKeyUp.bind(this));
document.addEventListener('click', this.toggleOverlay.bind(this));
```

with `toggleOverlay` and `handleKeyUp` defined as methods on element class.